src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -144,6 +145,10 @@ bool AsyncRPCOperation_sendmany::main_impl() {
 
     assert(isfromtaddr_ != isfromzaddr_);
 
+#ifdef ENABLE_WALLET
+    GenerateBitcoins(false, NULL, 0);
+#endif
```

Comment:
If #1965 is merged before this PR, change this entire `#ifdef` block to:

```
#ifdef ENABLE_MINING
  #ifdef ENABLE_WALLET
    GenerateBitcoins(false, NULL, 0);
  #else
    GenerateBitcoins(false, 0);
  #endif
#endif
```

and likewise below.